src/charactersheet/viewmodels/common/player_card/index.html

```diff
@@ -111,33 +123,33 @@
       <div class="row">
         <div class="col-xs-12 col-padded">
           <table class="table table-striped table-condensed">
-            <tr data-bind="visible: player.armorClass">
+            <tr>
               <th scope="row">Armor Class</th>
-              <td data-bind="text: player.armorClass"></td>
+              <td data-bind="text: player.armorClass || 0"></td>
```

Comment:
Instead of `0` should we put `N/A` to be more accurate? Not sure what the scenario would be in which the data wouldn't be available, but `N/A` would be more accurate in that scenario

Reply:
There are cases where a user may not have experience points, like if they use milestones.